arbeitszeit_web/presenters/register_member_presenter.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases.register_member import RegisterMemberUseCase
+from arbeitszeit_web.register_member import RegisterForm
+from arbeitszeit_web.session import Session
+from arbeitszeit_web.translator import Translator
+
+
+@dataclass
+class RegisterMemberViewModel:
+    is_success_view: bool
+
+
+@dataclass
+class RegisterMemberPresenter:
+    session: Session
+    translator: Translator
+
+    def present_member_registration(
+        self, response: RegisterMemberUseCase.Response, form: RegisterForm
+    ) -> RegisterMemberViewModel:
+        if response.is_rejected:
+            if (
+                response.rejection_reason
+                == RegisterMemberUseCase.Response.RejectionReason.member_already_exists
+            ):
+                form.add_email_error(
```

Comment:
wouldn't it be more consistent if we just use our notification system (`notifier.display_warning()`) in this case instead of adding an error and iterating over the lists of errors in the template? of course we would lose the relationship with a specific input field, but the registration field has only 3 inputs.

Reply:
We could move the notification to the `Notifier`. We should keep in mind that we might want to do validation on the other fields as well, e.g. password strength validation. This means that we could "regret" making this message a global notification later down the line.